Resources/Computing/GlobusComputingElement.py

```diff
@@ -0,0 +1,283 @@
+########################################################################
+# File :   GlobusComputingElement.py
+# Author : A.S.
+########################################################################
+
+""" Globus Computing Element
+
+   Allows direct submission to Globus Computing Elements with a SiteDirector Agent
+
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC.Resources.Computing.ComputingElement          import ComputingElement
+from DIRAC.Core.Utilities.Grid                           import executeGridCommand
+from DIRAC                                               import S_OK, S_ERROR
+
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry   import getGroupOption
+from DIRAC.FrameworkSystem.Client.ProxyManagerClient     import gProxyManager
+from DIRAC.WorkloadManagementSystem.DB.PilotAgentsDB     import PilotAgentsDB
+
+
+import os, tempfile
+
+CE_NAME = 'Globus'
+MANDATORY_PARAMETERS = [ 'Queue' ]
+
+class GlobusComputingElement( ComputingElement ):
+  """Globus computing element class
+  implementing the functions jobSubmit, getJobOutput """
+
+  #############################################################################
+  def __init__( self, ceUniqueID ):
+    """ Standard constructor.
+    """
+    ComputingElement.__init__( self, ceUniqueID )
+
+    self.ceType = CE_NAME
+    self.submittedJobs = 0
+    self.mandatoryParameters = MANDATORY_PARAMETERS
+    self.pilotProxy = ''
+    self.queue = ''
+    self.outputURL = 'gsiftp://localhost'
+    self.gridEnv = ''
+    self.proxyRenewal = 0
+
+  #############################################################################
+  def _addCEConfigDefaults( self ):
```

Comment:
if you are not doing anything more that the parent class, you don't need to re-implement it